_studio/mfx_lib/shared/include/mfx_interface_scheduler.h

```diff
@@ -120,7 +120,8 @@ class MFXIScheduler : public MFXIUnknown
     virtual
     mfxStatus AdjustPerformance(const mfxSchedulerMessage message) = 0;
 
-
+    virtual
+    mfxStatus SetThreadNum(const void *pComponent, mfxU32 threadNum) = 0;
```

Comment:
@onabiull : that's the major question as of now:
1. Can we add method here?
2. Or we need MFXIScheduler3?
3. Or we can add to MFXIScheduler2?

Let's nail this down...

Reply:
Right now and Linux only we can. But considering future changes I'd stick to option 2 - adding new interface.